src/shared/lib/URLWrapper.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { assert } from "chai";
+import ResultsViewURLWrapper from "pages/resultsView/ResultsViewURLWrapper";
+import { autorun, observable, reaction } from "mobx";
+import ExtendedRouterStore from "shared/lib/ExtendedRouterStore";
+import sinon from "sinon";
+
+describe("URLWrapper", () => {
```

Comment:
this is really ResultsViewURLWRapper.spec.ts

Reply:
I think it might be ok to test a superclass using it's subclass